build-tools/xaprepare/xaprepare/Steps/Step_ThirdPartyNotices.cs

```diff
@@ -172,7 +172,7 @@ void ProcessTPN (SortedDictionary <string, ThirdPartyNotice> licenses, ThirdPart
 			Log.StatusLine ($"  {Context.Instance.Characters.Bullet} Processing: ", tpn.Name, ConsoleColor.Gray, ConsoleColor.White);
 
 			if (licenses.ContainsKey (tpn.Name)) {
-				Log.WarningLine ($"Duplicate Third Party Notice '{tpn.Name}' (old class: {licenses [tpn.Name]}; new class: {tpn})");
+				Log.InfoLine ($"Duplicate Third Party Notice '{tpn.Name}' (old class: {licenses [tpn.Name]}; new class: {tpn})");
```

Comment:
@grendello does this represent some issue that we should try to fix rather than suppressing the warning?

Reply:
The intention was that if we see this warning, we go in and remove the duplicate, and the assumption was that it would be a very rare occurrence.  So, yep, if we see it, we should remove the license duplicates :)

Reply:
The warning is caused by the TPN `Irony`, which is used by both `Xamarin.Android.Build.Tasks` and `Xamarin.Android.Tools.Aidl`.  I considered removing one of the instances but I was afraid that might cause us to miss the TPN in the future.

For example, we remove the `Xamarin.Android.Tools.Aidl` version today.  Then in the future we remove the need for `Xamarin.Android.Build.Tasks` to use `Irony`, so we remove that version.  We would not remember that we need to add the `...Aidl` version back.

It felt safer to keep both and reduce the warning than risk accidentally removing them both in the future.

But I'm open to other suggestions.